### PR TITLE
fix: #397

### LINKED ISSUE #397: The gatekeeper-operator namespace collision
**Describe the bug:** (a clear and concise description of what the bug is)
I can't deploy otomi from scratch

**To Reproduce**

1. Create brand new cluster
2. Enable `gatekeeper-operator` chart `charts.gatekeeper-operator.enabled: true`
3. `otomi deploy`
4. Observe that `gatekeeper-operator` tries to create namespace 
5. Observe the following error:
```
Release "gatekeeper-operator" does not exist. Installing it now.
in helmfile.d/helmfile-03.init.yaml: failed processing release gatekeeper-operator: command "/home/app/tools/helm" exited with non-zero status:
PATH:
  /home/app/tools/helm
ARGS:
  0: helm (4 bytes)
  1: upgrade (7 bytes)
  2: --install (9 bytes)
  3: --reset-values (14 bytes)
  4: gatekeeper-operator (19 bytes)
  5: ../charts/gatekeeper-operator (29 bytes)
  6: --wait (6 bytes)
  7: --timeout (9 bytes)
  8: 1200s (5 bytes)
  9: --atomic (8 bytes)
  10: --create-namespace (18 bytes)
  11: --namespace (11 bytes)
  12: default (7 bytes)
  13: --values (8 bytes)
  14: /tmp/helmfile611285305/default-gatekeeper-operator-values-58d96d5d7d (68 bytes)
  15: --values (8 bytes)
  16: /tmp/helmfile667484996/default-gatekeeper-operator-values-84575784b4 (68 bytes)
  17: --history-max (13 bytes)
  18: 3 (1 bytes)
ERROR:
  exit status 1
EXIT STATUS
  1
STDERR:
  Error: rendered manifests contain a resource that already exists. Unable to continue with install: Namespace "gatekeeper-system" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "gatekeeper-operator"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "default"
COMBINED OUTPUT:
  Release "gatekeeper-operator" does not exist. Installing it now.
  Error: rendered manifests contain a resource that already exists. Unable to continue with install: Namespace "gatekeeper-system" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "gatekeeper-operator"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "default"
```
**Expected behavior:** (a clear and concise description of what you expected to happen)

The `gatekeeper-operator` does not create namespace.

### PR DESCRIPTION
**Pull request by Morriz**
fixes #397

### CODE REVIEW

**Review by j-zimnowoda — APPROVED** [review]
Yes, that change should be sufficient to solve the issue. Thank you.

File: helmfile.d/snippets/defaults.gotmpl [hunk 1]
```diff
@@ -6,7 +6,6 @@ helmDefaults:
   force: false
   cleanupOnFail: false
   skipDeps: true
-  disableValidation: true
```
Comment (Dunky13) [inline helmfile.d/snippets/defaults.gotmpl:9]: Why do we `disableValidation`?

**Review by Dunky13 — COMMENTED** [review]
One question, rest looks good

File: helmfile.d/snippets/defaults.gotmpl (hunk 1 above)
Comment (j-zimnowoda) [reply helmfile.d/snippets/defaults.gotmpl:9]: This line has been removed in this PR

**Review by Dunky13 — APPROVED** [review]

**Pull request merged**